src/pages/index.js

```diff
@@ -52,28 +100,54 @@ cardSection.renderItems();
 const editProfilePopup = new PopupWithForm(
   '#profile-edit-modal',
   (formData) => {
-    userInfo.setUserInfo({
-      name: formData.title,
-      description: formData.description
-    });
+    api.setUserInfo(formData.title, formData.description)
```

Comment:
* When I add a new card, edit the profile data or change the avatar, there is an error https://snipboard.io/boucEA.jpg 


Please, add `return` here to be able to use `.then`, `.catch`., `.finally` inside the `PopupWithForm` class

```js
  (formData) => {
    return api  // here
      .setUserInfo(formData.title, formData.description)
```